app/models/commissioning_document_template/mappa.rb

```diff
@@ -8,6 +8,7 @@ def context
       super.merge(
         aliases: kase.subject_aliases,
         pnc: kase.other_subject_ids,
+        date_range: data_request.request_dates.capitalize,
```

Comment:
The screenshot on the ticket doesn't have the first character capitalised
```suggestion
        date_range: data_request.request_dates,
```